src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -4179,7 +4185,7 @@ public void forceKeyspaceFlush(String keyspaceName, String... tableNames) throws
         for (ColumnFamilyStore cfStore : getValidColumnFamilies(true, false, keyspaceName, tableNames))
         {
             logger.debug("Forcing flush on keyspace {}, CF {}", keyspaceName, cfStore.name);
-            cfStore.forceBlockingFlush();
+            cfStore.forceBlockingFlush(ColumnFamilyStore.FlushReason.USER_FORCED);
```

Comment:
I think there are also some cases, like in `RepairTest`, where we call this but could make an argument this should be `UNIT_TESTS`.